daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md

```diff
@@ -122,6 +125,31 @@ For example, if installing using the example above, the RabbitMQ server client a
 
 {{< /tabs >}}
 
+## Use topic exchange to route messages
+Setting `exchangeKind` to `"topic"` will use the topic exchanges, which are commonly used for the multicast routing of messages. 
+Messages with routing key will be routed to one or many queues based on the routing key defined in the netadata when subscribing.
```

Comment:
```suggestion
Messages with a routing key will be routed to one or many queues based on the routing key defined in the metadata when subscribing.
```

Reply:
Where is the routing key defined? I Could not find this? Can you put a comment on where/how to set the routing key?

Reply:
Done.